src/components/Intents/IntentHandler.jsx

```diff
@@ -0,0 +1,93 @@
+import React, { Children, Component } from "react";
```

Comment:
Looks like my previous comments disappeared — this file is largely borrowed from cozy-store. It allows for a nice intent handling in the app.